tsconfig.json

```diff
@@ -45,7 +45,7 @@
     "noFallthroughCasesInSwitch": true /* Report errors for fallthrough cases in switch statement. */,
     "noUncheckedIndexedAccess": true /* Include 'undefined' in index signature results */,
     // "noPropertyAccessFromIndexSignature": true,  /* Require undeclared properties from index signatures to use element accesses. */
-    "importsNotUsedAsValues": "error",
+    // "importsNotUsedAsValues": "error",
```

Comment:
Need to solve this before merge.

Reply:
Replaced with updated value -> https://github.com/microsoft/TypeScript/pull/52203